user/contactmethod/store.go

```diff
@@ -4,166 +4,61 @@ import (
 	"context"
 	"database/sql"
 	"encoding/json"
-	"time"
 
+	"github.com/google/uuid"
+	"github.com/target/goalert/gadb"
 	"github.com/target/goalert/permission"
-	"github.com/target/goalert/util"
 	"github.com/target/goalert/util/log"
-	"github.com/target/goalert/util/sqlutil"
 	"github.com/target/goalert/validation"
 	"github.com/target/goalert/validation/validate"
 )
 
 // Store implements the lookup and management of ContactMethods against a *sql.Store backend.
 type Store struct {
 	db *sql.DB
```

Comment:
Let's delete this to ensure it's not being used (since we should be passing the dbtx in)

Reply:
removing this field conflicts with the `FindMany` func as adding a new parameter `dbtx` messes with the implementation of 
```ctx = context.WithValue(ctx, dataLoaderKeyCM, dataloader.NewStoreLoader(ctx, a.CMStore.FindMany))```